### DISCUSSION TITLE
Alpha / Beta nodes

### DISCUSSION THREAD

**Discussion by Durman** [nortikin/sverchok#4685] [Ideas]
There are Alpha and Beta categories now. But they are not explicit enough to give users information about stage of a node. Because a node can be added via search operator which does not give information about the node state.

Solution is to mark nodes by frames similar how it works with deprecated nodes. I guess all new nodes should be marked as beta until a next release.

![image](https://user-images.githubusercontent.com/28003269/192459050-eef388f7-b3d8-4078-bcd0-5de83e6583f8.png)

**Comment by portnov** [comment]
Good idea, in general.
But what to do with existing categories? Move all nodes from them to other categories?

**Reply by Durman** [reply]
Yes, I don't see a reason to keep them there with new approach.

**Comment by Durman** [comment]
Done #4697
